app/static/css/style.css

```diff
@@ -5,6 +5,11 @@
     padding-bottom: 20px;
 }
 
+#in{
```

Comment:
ID name is not meaningful, please use some meaningful ID name.

Reply:
I want to write info but it was there in default so I used in